lmdeploy/serve/vl_async_engine.py

```diff
@@ -61,9 +61,10 @@ async def _get_prompt_input(self,
         results = {}
         input_ids = []
         if len(segs) > 1:
-            images = await self.vl_prompt_template.async_collect_pil_images(
+            images_with_kwargs = await self.vl_prompt_template.async_collect_pil_images(  # noqa: E501
```

Comment:
may put `noqa` at the end of a full line like:
`images_with_kwargs = await self.vl_prompt_template.async_collect_pil_images(prompt)  # noqa: E501`